Source/Core/Core/IOS/ES/TitleManagement.cpp

```diff
@@ -394,19 +402,25 @@ IPCCommandResult ES::ImportTitleDone(Context& context, const IOCtlVRequest& requ
 
 ReturnCode ES::ImportTitleCancel(Context& context)
 {
-  if (!context.title_import.tmd.IsValid())
+  // It is possible to get into a state where the TMD buffer exists and
+  // with the title import not being considered valid.
+  if (context.title_import.tmd.GetBytes().empty() || context.title_import.content.valid)
```

Comment:
No, because we want to prevent the title import from being cancelled if there's still a content import in progress. Would you like me to mention this in the comment?